### PR TITLE
[SPARK-14306][ML][PySpark] PySpark ml.classification OneVsRest support export/import

### PR DESCRIPTION
**Pull request by yinxusen**
## What changes were proposed in this pull request?

https://issues.apache.org/jira/browse/SPARK-14306

Add PySpark OneVsRest save/load supports.
## How was this patch tested?

Test with Python unit test.

### CODE REVIEW

**Comment by yinxusen** [comment]
test it please

**Comment by yinxusen** [comment]
ping @jkbradley

**Comment by jkbradley** [comment]
Thanks!  I'll take a look now

File: python/pyspark/ml/classification.py [hunk 1]
```diff
@@ -1160,8 +1160,33 @@ def weights(self):
         return self._call_java("weights")
 
 
+class OneVsRestParam(HasFeaturesCol, HasLabelCol, HasPredictionCol):
```
Comment (jkbradley) [inline python/pyspark/ml/classification.py]: name should match other Params classes: "OneVsRestParams"

**Comment by jkbradley** [comment]
Nice PR, this looks good except for the 1 nit I commented on.

After this PR, I believe we will have full Python coverage for persistence!  I'd like to do something of an integration test for persistence in ml/tests.py in the "PersistenceTest" suite.  I just made a JIRA: [https://issues.apache.org/jira/browse/SPARK-14706].  Would you be interested in working on that one?

**Comment by yinxusen** [comment]
@jkbradley Sure, I can do that. I'll fix this one first.

**Comment by yinxusen** [comment]
test it please

**Comment by jkbradley** [comment]
LGTM pending tests

**Comment by jkbradley** [comment]
Merging with master.  Thanks for the PR!

**Pull request closed without merging**